Library/Homebrew/cask/utils/xattr.swift

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/env swift
```

Comment:
This is a lot of Swift to add to a codebase where the vast majority of the maintainers have never written any Swift. I'm a strong 👎 due to that, sorry.